crates/bevy_tasks/src/single_threaded_task_pool.rs

```diff
@@ -14,8 +15,19 @@ impl TaskPoolBuilder {
         Self::default()
     }
 
-    /// No op on the single threaded task pool
-    pub fn num_threads(self, _num_threads: usize) -> Self {
+    /// Override the number of compute-priority threads created for the pool. If unset, this default to the number
```

Comment:
should this comment and the two below be more in line of "No op on the single threaded task pool"?